submissions/lolaOluwadahunsi/htmlForms/contact_form_page.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Contact Form</title>
+</head>
+<body>
+<h1>Contact Form</h1>
+<form action="#" method="POST" enctype="application/x-www-form-urlencoded">
```

Comment:
@Lolatech07 Remove attributes `action`, `method` and `enctype`. They are not needed for now.